com.yubico.YubiOath.json

```diff
@@ -0,0 +1,258 @@
+	},
+	{
+            "name": "pycparser",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/eliben/pycparser/archive/release_v2.19.tar.gz",
+		    "sha256": "893ec3d0f26c0faa7d0b992979940353a1682f86b5cd1228740637055b112379"
+		}
+	    ]
+	},
+	{
+            "name": "cryptography",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"python3 setup.py install --prefix=/app"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pyca/cryptography/archive/2.7.tar.gz",
+		    "sha256": "6e01aafcd430086f1a943dcf04d71cfc2247aedeec5d6c474ce5450acd2fbdd7"
+		}
+	    ]
+	},
+	{
+            "name": "fido2",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/Yubico/python-fido2/releases/download/0.7.0/fido2-0.7.0.tar.gz",
+		    "sha256": "47b02852780849bb4bb698b9727d61970ee77a83eb25715fe7c6235ebd648d87"
+		}
+	    ]
+	},
+	{
+            "name": "pyopenssl",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pyca/pyopenssl/archive/19.0.0.tar.gz",
+		    "sha256": "c5689bb12c210e3f996ee0a49cee8867f2afbf7e9beb84f3522f9dc8fdcb83ae"
+		}
+	    ]
+	},
+	{
+            "name": "yubikey-manager",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/Yubico/yubikey-manager/archive/yubikey-manager-3.1.0.tar.gz",
+		    "sha256": "9009ad6695b262c23abfb2d030786bcb5d72756f822ce74eb20d611a9cd130bd"
+		}
+	    ]
+	},
+        {
+            "name": "yubioath-desktop",
+            "buildsystem": "qmake",
+	    "post-install": [
+		"sed -i 's/Icon=yubioath/Icon=com.yubico.YubiOath/' resources/yubioath-desktop.desktop",
+		"install -Dm644 resources/yubioath-desktop.desktop /app/share/applications/com.yubico.YubiOath.desktop",
+		"install -Dm644 resources/icons/yubioath.png /app/share/icons/hicolor/128x128/apps/com.yubico.YubiOath.png",
+                "install -Dm644 com.yubico.YubiOath.appdata.xml /app/share/appdata/com.yubico.YubiOath.appdata.xml"
+	    ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://developers.yubico.com/yubioath-desktop/Releases/yubioath-desktop-4.3.6.tar.gz",
+		    "sha256": "451d47f23d08dbe3630dcaa14bf11bbd8e894c86e9dee6264af597b5a4550468"
+                },
+		{
+		    "type": "patch",
+		    "path": "yubioath-desktop-path.patch"
+		},
+                {
+		    "type": "file",
+		    "path": "com.yubico.YubiOath.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html